controllers/chat/getSignedChannelList.js

```diff
@@ -128,7 +136,7 @@ const getSignedChannelList = async (req, res) => {
     const channels = [];
     for (let i = 0; i < queriedChannels.length; i++) {
       const _channelItem = queriedChannels[i];
-      const _filteredChannel = await __filterAndTransformChannelData(_channelItem);
+      const _filteredChannel = await __filterAndTransformChannelData(_channelItem, userId);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `__filterAndTransformChannelData` is now being called with the additional `userId` parameter. This change is consistent with the updated function definition.
```diff
-      const _filteredChannel = await __filterAndTransformChannelData(_channelItem);
+      const _filteredChannel = await __filterAndTransformChannelData(_channelItem, userId);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->